### ISSUE TITLE
Add desktop support

### ISSUE THREAD

**Issue by prateekmedia** [newtaDev/pod_player#29]
I know saying is easier than done but due to the nature of flutter if you can use dart_vlc as a desktop replacement using platform interface then it would be good, also dart_vlc let's you customize the player controls playback speed, source, etc, etc.

**Comment by newtaDev** [comment]
Thanks @prateekmedia  for letting me know about this awesome plugin `dart_vlc`
I will definitely give it a try

But this plugin doesn't provide support for Android and web platform? 
So if we try to implement this plugin, doesn't it effect Android and web platforms..?

**Comment by prateekmedia** [comment]
@newtaDev Nope, there is something called as platform interface so you will have to divide this repk into two folders pod_player and pod_player_platform_interface. If you want you can also create pod_player_desktop folder.

It will have the logic that for which platform what plugin dependency and code has to be used

You can see how just_player does it.

**Comment by newtaDev** [comment]
> @newtaDev Nope, there is something called as platform interface so you will have to divide this repk into two folders pod_player and pod_player_platform_interface. If you want you can also create pod_player_desktop folder.
> 
> It will have the logic that for which platform what plugin dependency and code has to be used
> 
> You can see how just_player does it.

Thank you for sharing.
I will try to implement
